build/ci/release.yml

```diff
@@ -216,7 +216,34 @@ functions:
           - tool_name
         working_dir: src/github.com/mongodb/mongodb-atlas-cli/build/package/docker/meta
         binary: ./docker-build.sh
-
+  "generate evergreen":
+    - command: subprocess.exec
+      type: test
+      params:
+        add_to_path:
+          - ${go_bin}
+        include_expansions_in_env:
+          - tool_name
+          - workdir
+        working_dir: src/github.com/mongodb/mongodb-atlas-cli
+        env:
+          <<: *go_options
+        binary: build/ci/evergreen-generate-tasks.sh
+    - command: generate.tasks
+      params:
+        files:
+          - tasks.json
```

Comment:
[nit] is usually a good practice to attach the generated json to the task to be able to debug it, you can use evg s3 commands for this

Reply:
done!